geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/stats/StatsHistogramCommand.scala

```diff
@@ -276,7 +272,7 @@ object StatsHistogramCommand {
     """                         ,,,________,.:__.,.,,:: ,                 .:,________________________________,,,__________,,                       """,
     """                          __________.,.: .,__,/:                    __.,,,____,:;__.:,.,;__.,:,__________________,, :                       """,
     """                          .,__________,__.,                      ,,.:  : , .::;.,:.,__:,.,__________________,:__   __                       """,
-    """                           ,println(,,__________.,,                       :,,__.      :,;__________________________,,  ,.:,,                        """,
```

Comment:
yeah, that 'println' got inserted at some point and we didn't notice... probably a find/replace gone awry